### ISSUE TITLE
Choosing "New" on system menu with mouse broken on Cygwin since 2.2.1

### ISSUE THREAD

**Issue by jon-turney** [mintty/mintty#491]
Selecting "New" from the sysmenu to create a new mintty window does nothing in 2.2.1

Oddly the Alt+F2 shortcut still seems to work, though...

**Comment by mintty** [comment]
Works for me.
Maybe you are trying with a window that has explicit position parameters? (The new window would be placed on the same position...)
Otherwise please provide more environment information (Windows version, start mode (shortcut?)...).

**Comment by jon-turney** [comment]
This is on Windows 8.1 x86_64.

The problem seems to occur independently of how I start mintty (from shortcut, from windows command prompt, from a terminal)

It seems the only system I can reproduce this on is my multimonitor one, and on a bit more investigation, it does seem related to f46f10d somehow.

```
$ gdb --args ./mintty
GNU gdb (GDB) Cygwin 7.9.1-1
[...]
Reading symbols from ./mintty...done.
(gdb) b child_fork
Breakpoint 1 at 0x1004091ea: file child.c, line 514.
(gdb) r
Starting program: /wip/mintty/bin/cygwin64/mintty
warning: Application "\??\C:\cygwin64\wip\mintty\bin\cygwin64\mintty.exe" found in cache
[use 'New' from sysmenu in child mintty]
Breakpoint 1, child_fork (argc=1, argv=0x23cb20, moni=17760913) at child.c:514
514       pid_t clone = fork();
(gdb) bt
#0  child_fork (argc=1, argv=0x23cb20, moni=17760913) at child.c:514
#1  0x0000000100419f0e in win_proc (wnd=0xe0d04, message=274, wp=192, lp=17760945) at winmain.c:993
#2  0x00007ff80fbf3ff0 in USER32!CallWindowProcW () from /cygdrive/c/Windows/system32/USER32.dll
#3  0x00007ff80fbf2367 in USER32!NotifyWinEvent () from /cygdrive/c/Windows/system32/USER32.dll
#4  0x000000010041cc4c in main (argc=1, argv=0x60003c0d0) at winmain.c:1966
(gdb)
```
I don't really see how this is supposed to work.  [MSDN] (https://msdn.microsoft.com/en-us/library/windows/desktop/ms646360%28v=vs.85%29.aspx) says that if the sysmenu item is chosen with the mouse, LPARAM contains the mouse coordinates, but that commit seems to want to use it to pass the monitor number...

**Comment by mintty** [comment]
Fixed in 2.2.2.

**Comment by jon-turney** [comment]
Confirmed this is fixed.  Thanks.
